testing/iframe.js

```diff
@@ -275,11 +275,15 @@ const IFRAME_STUB_URL =
  * See /test/fixtures/served/iframe-stub.html for implementation.
  *
  * @param win {!Window}
+ * @param opt_beforeAttachToDom {function=}
```

Comment:
LGTM

Regards,

Hongfei

On Fri, Aug 19, 2016 at 3:09 PM, Hongfei Ding <notifications@github.com>
wrote:

> In testing/iframe.js
> <https://github.com/ampproject/amphtml/pull/4634#discussion_r75558547>:
>
> > @@ -275,11 +275,15 @@ const IFRAME_STUB_URL =
> >   * See /test/fixtures/served/iframe-stub.html for implementation.
> >   *
> >   * @param win {!Window}
> > + * @param opt_beforeAttachToDom {function=}
>
> line 285 calls the function with one param. that means we expect a
> function with one param.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4634/files/81cd25677abbdd27582a65675cb7ff3b7bbb1ef8#r75558547>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANd2kAxXWnznydGpitzPYhVHHP8y8x9nks5qhimVgaJpZM4JowqV>
> .
>
> --
> You received this message because you are subscribed to the Google Groups
> "amphtml-eng-github" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to amphtml-eng-github+unsubscribe@google.com.
> To post to this group, send email to amphtml-eng-github@google.com.
> To view this discussion on the web visit https://groups.google.com/a/
> google.com/d/msgid/amphtml-eng-github/ampproject/amphtml/
> pull/4634/r75558547%40github.com
> <https://groups.google.com/a/google.com/d/msgid/amphtml-eng-github/ampproject/amphtml/pull/4634/r75558547%40github.com?utm_medium=email&utm_source=footer>
> .
>